lib/Extension/Rpc/Handler/AbstractHandler.php

```diff
@@ -10,23 +10,26 @@
 
 abstract class AbstractHandler implements Handler
 {
-    private $requiredArguments = [];
+    /** @var array<string, Input> */
+    private array $requiredArguments = [];
 
     protected function requireInput(Input $input): void
     {
         $this->requiredArguments[$input->name()] = $input;
     }
 
-    protected function hasMissingArguments(array $arguments)
+    /** @param array<bool|float|int|string|null> $arguments */
```

Comment:
there is a `scalar` type (which I've never used) and is defined in PHPStan as:

```
				return new UnionType([new IntegerType(), new FloatType(), new StringType(), new BooleanType()]);
```

Reply:
Yeah it's basically mixed but without the object. (arrays are possible values too)

Reply:
so `array<scalar|array<scalar>|null>` ?